sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -1056,6 +1057,11 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
       call read_netcdf_2d_para(ncid3d,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
       spval,VarName,rel_vort_maxhy1(ista_2l,jsta_2l))
      if(debugprint)print*,'sample ',VarName,' =',rel_vort_maxhy1(isa,jsa)
+! HAILCAST maximum hail diameter (mm) since last output
+      VarName='HAILCAST_DHAIL'
```

Comment:
@jaymes-kenyon May I ask why this variable name is in upper case? Most of variables from the UFS based models are in lower case.

Reply:
Hi @WenMeng-NOAA — I noticed this, too.  Looking in the dyn*.nc files, this seems to be the only array that uses upper case (see screenshot).  I tried  coding `INITPOST_NETCDF.f` with "**hailcast_dhail**" (lower case), but the array was not read-in correctly.
<img width="559" alt="Screenshot 2023-10-25 at 8 34 59 AM" src="https://github.com/NOAA-EMC/UPP/assets/14337181/c1dea480-b8f0-44ff-bcb0-5e66f76e318a">

Reply:
@jaymes-kenyon Would you check the diag table "diag_table" in the UFS model side if this variable is set to be written out in netcdf file in upper case?

Reply:
@WenMeng-NOAA — Thanks!  I was able to modify the RRFS diag_table to produce a lower-case array name in the dyn*.nc files.  Updated *.nc files are now available on Hera: 
`/scratch1/BMC/wrfruc/kenyon/upp_development/hailcast/2022072009`
With my latest commit to this PR, I changed `INITPOST_NETCDF.f` to use this lower-case name.  Thanks for the suggestion and the help!

Reply:
@jaymes-kenyon Thanks!